joomla/joomla-cms
# Fix the read more std-button

Pull Request for Issue #15862 .

### Summary of Changes
Readmore is targeted to parent window and it should be in the same (there is no modal). My bad :(


### Testing Instructions
```
Create new Menu Item
Choose "Single Article" type
Click "Create" button for article
Click "Read More" button

Also check that in normally created articles the button works correctly 
```


### Expected result



### Actual result



### Documentation Changes Required
